package_maximizer/adapters/__init__.py

```
@@ -140,6 +173,21 @@ def parse_multi(self, raw: str) -> list[PackageMetadata]:
 class PipMetadataAdapter:
     """Парсер метаданных pip (pip show / METADATA)."""
 
+    def fetch(self, package_name: str) -> PackageMetadata | None:
+        """Получить метаданные через pip show."""
+        try:
+            result = subprocess.run(
+                ["pip", "show", package_name],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)